src/pages/tutorials/aiops/installation/index.mdx

```diff
@@ -70,36 +69,62 @@ You can check your entitlement for Watson AIOPs by clicking **View Library**
 
 ![](images/entitlement_check.png)
 
-**NOTE:** If you do not have an existing IBM contract that allows you access to the software for these labs, you can register for trial using link [Trial license of Watson AIOps](https://www.ibm.com/account/reg/us-en/login?formid=urx-51074)
+**NOTE:** If your entitlement key is not valid for Watson for AIOps Event Manager, you can register for a 60 day trial using the following link: [Request Watson AIOps trial](https://www.ibm.com/account/reg/us-en/login?formid=urx-51074).
 
 ## 2-4: Prepare your OpenShift cluster
 
-This step involves creating a namespace to install the Watson for AIOps Event Manager Operator into, and setting up your software entitlement key, so that you will be able to deploy an instance of Event Manager.
+This step involves creating a namespace (also known as a "Project" on OpenShift) to install the Watson for AIOps Event Manager Operator into, and setting up your software entitlement key, so that you will be able to deploy an instance of Event Manager.
 
-This step involves the following tasks:
+This section involves the following tasks:
 
-- Open the **OpenShift UI**
-- Acquire a token and log in to the OpenShift cluster via the command line
+- Navigate to and open the **OpenShift UI**
+- Establish a connection to your control virtual machine using the supplied SSH key
+- Acquire an OpenShift login token and log in to the OpenShift cluster via the command line
 - Create a **namespace** to deploy Event Manager into
 - Create an **entitlement key secret**
 
-Go to your newly created OpenShift cluster and click OpenShift web console to open the main OpenShift UI:
+During this module, you will need to log in to your dedicated "control" virtual machine, to run configuration commands against your OpenShift cluster.
+
+Prior to starting this lab, you should have received two emails: one containing the IP address of your "control" virtual machine, and one containing a zip file with an id_rsa private and public key, which you will need to access the control VM.
+
+- Follow the instructions in the email to unzip the id_rsa.zip file using the supplied password and save the key files onto your laptop
+- Open a command terminal (on Windows, use "Windows Powershell")
+- In the terminal session, navigate to the directory where your rsa_id file is located
+- Open an SSH session as the "jammer" user to your control box using the IP address provided to you in the other email - example:
```

Comment:
We should remove the mention of the emails. The instruction to open an SSH session to their control node is cool. This will map to the arch diagram we're pushing up into the getting started section. So even if a user comes to this repo outside a tech jam event, they can provision their own infra and get going.